api/cmd/portainer/main.go

```diff
@@ -237,6 +238,17 @@ func updateSettingsFromFlags(dataStore portainer.DataStore, flags *portainer.CLI
 	return nil
 }
 
+func overrideSettingsFromFlags(dataStore portainer.DataStore, flags *portainer.CLIFlags) error {
+	settings, err := dataStore.Settings().Settings()
+	if err != nil {
+		return err
+	}
+
+	settings.EnableOpenAMTFeatures = *flags.EnableOpenAMTFeatures
+
+	return dataStore.Settings().UpdateSettings(settings)
+}
+
```

Comment:
I wonder if the updateSettingsFromFlags and overrideSettingsFromFlags could be merged.
They do very similar things.   What do you think?

Reply:
i think i may have started that in the store refactor - I'd suggest tat woudl be a different PR

Reply:
Sounds good.

Reply:
I just did it like that to avoid messing with the other settings, that apparently can only be set at DB creation time (!?).